src/util/match-route-with-pattern.ts

```diff
@@ -8,9 +9,12 @@ import matchstick from "matchstick";
  *
  */
 export const matchRouteWithPattern = (pattern: string, route: string): boolean => {
-	pattern = pattern.replace(/ac\./gm, ""); // Remove all the `ac.`
+	if (isEmpty(pattern) || isEmpty(route)) {
+		return false;
+	}
+	pattern = pattern.replace("ac.", ""); // Remove all the `ac.`
```

Comment:
**BLOCKING**
Somehow in chrome console it says otherwise. And we do have some line with multiple "ac." `url:"/jira/connect/enterprise/{ac.serverUrl}/app?new={ac.new}",`
![image](https://user-images.githubusercontent.com/105693507/185003993-57d6a860-dc43-47bd-b070-f99b0c261744.png)



Maybe revert this change ( as it is not part of the PR)? Or try to test it more.

Reply:
Huh, the tests did pass but I guess this scenario wasn't tested.  Added said scenario.